app/blockchain/fabric-samples/asset-transfer-basic/chaincode-typescript/src/asset.ts

```
@@ -0,0 +1,54 @@
+/*
+  SPDX-License-Identifier: Apache-2.0
+*/
+
+import { Object, Property } from 'fabric-contract-api';
```

Biome (2.1.2)

[error] Do not shadow the global "Object" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)